features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/advanced/AdvancedSettingsView.kt

```diff
@@ -81,6 +81,18 @@ fun AdvancedSettingsView(
                 onChange = { state.eventSink(AdvancedSettingsEvents.SetDeveloperModeEnabled(it)) },
             ),
         )
+        ListItem(
+            headlineContent = {
+                Text(text = "Send read receipts")
+            },
+            supportingContent = {
+                Text(text = "On disabling, others won't see your read receipt.")
```

Comment:
```suggestion
                Text(text = "On disabling, others won't see your read receipts.")
```

Also I think those 2 strings should be defined in Localazy.